src/cephadm/cephadm

```diff
@@ -787,6 +787,10 @@ class CephIscsi(object):
         # remove extra container args for tcmu container.
         # extra args could cause issue with forking service type
         tcmu_container.container_args = []
+        # set container limits to unlimited as defaults (Docker 4096 / Podman 2048)
+        # prevents the creation of max lun (default 255)
+        pids_unlimited = '-1' if isinstance(self.ctx.container_engine, Podman) else '0'
+        tcmu_container.container_args.extend(['--pids-limit=%s' % pids_unlimited])
```

Comment:
IIUC this PR lifts the limit for all containers, unconditionally.  Why is this iSCSI-specific bit still present?  Wouldn't the extra argument be already appended by `get_container()` which is called a few lines above?

Reply:
tcmu-runner container is not managed by systemd (seems to be special case) and get_container() does not apply on it. The CephIscsi object in get_container() is about rbd-target-api only

Reply:
Well, `get_container()` is still called and it seems to append `--pids-limit` unconditionally.  However, looking closer, `get_tcmu_runner_container()` then strips all appended arguments:
```
        # remove extra container args for tcmu container.
        # extra args could cause issue with forking service type
        tcmu_container.container_args = []
```
So yeah, the way this is coded, the iSCSI-specific bit appears to be necessary.

Reply:
> The CephIscsi object in get_container() is about rbd-target-api only

I would be nice to mention this in a comment.

Reply:
All good now (I hope)